### PR TITLE
Build and Release using Travis CI

### PR DESCRIPTION
**Pull request by abhinayagarwal**
* Releases snapshots on every commit to master branch
* Releases the current version when a tag is pushed

**N.B.** All environment variables have already been added to Travis

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,26 +1,15 @@
-# Compiled class file
-*.class
```
Comment (johanvos) [inline .gitignore]: why is this removed?

File: .ci/release.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cp .travis.settings.xml $HOME/.m2/settings.xml && mvn deploy -DskipTests=true -B -U -Prelease
+
+# Update version by 1
+newVersion=${TRAVIS_TAG%.*}.$((${TRAVIS_TAG##*.} + 1))
+
+# Replace first occurrence of TRAVIS_TAG with newVersion appended with SNAPSHOT
+sed -i "0,/<version>$TRAVIS_TAG/s//<version>$newVersion-SNAPSHOT/" pom.xml
+
+git commit pom.xml -m "Upgrade version to $newVersion-SNAPSHOT" --author "Github Bot <githubbot@gluonhq.com>"
+git push https://gluon-bot:$GITHUB_PASSWORD@github.com/gluonhq/javafx-maven-plugin HEAD:master
```
Comment (tiainen) [inline .ci/release.sh]: Shouldn't the github account name be `openjfx` instead of `gluonhq`?

File: .gitignore (hunk 1 above)
Comment (abhinayagarwal) [reply .gitignore]: It is covered by `target` directory for Maven projects.

File: .ci/release.sh (hunk 2 above)
Comment (abhinayagarwal) [reply .ci/release.sh]: Yup. I'll fix it.

Comment (abhinayagarwal) [reply .ci/release.sh]: Also, we need to make sure that Gluon bot has edit privileges for projects in OpenJFX.
